presto-parquet/src/test/java/com/facebook/presto/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -223,11 +223,13 @@ public void testString()
                 .withMessage("Corrupted statistics for column \"StringColumn\" in Parquet file \"testFile\": [min: taco, max: apple, num_nulls: 0]");
     }
 
-    private static BinaryStatistics stringColumnStats(String minimum, String maximum)
+    private static Statistics stringColumnStats(String minimum, String maximum)
     {
-        BinaryStatistics statistics = new BinaryStatistics();
-        statistics.setMinMax(Binary.fromString(minimum), Binary.fromString(maximum));
-        return statistics;
+        Statistics.Builder builder = Statistics.getBuilderForReading(new PrimitiveType(Type.Repetition.OPTIONAL, PrimitiveType.PrimitiveTypeName.BINARY, "testFile", OriginalType.UTF8));
```

Comment:
You can import `OPTIONAL`, `BINARY`, `UTF8`.

Reply:
Done.